### PR TITLE
Add a pointer to Android build doc and a pointer to Hello World sample

### PR DESCRIPTION
**Pull request by fanyang-mono**

### CODE REVIEW

**Comment by SamMonoRT** [comment]
LGTM !

File: docs/workflow/building/mono/README.md [hunk 1]
```diff
@@ -53,6 +53,10 @@ The build has a number of options that you can learn about using build -?.
 
 See the instructions in [../libraries/webassembly-instructions.md].
 
+### Android
+
+See the instructions in [../../testing/libraries/testing-android.md]
+
 ## Packages
```
Comment (EgorBo) [inline docs/workflow/building/mono/README.md:60]: Can you also add a section for iOS as well? 
doc: https://github.com/dotnet/runtime/blob/master/docs/workflow/testing/libraries/testing-apple.md

**Review by EgorBo — APPROVED** [review]

File: docs/workflow/building/mono/README.md [hunk 2]
```diff
@@ -53,6 +53,10 @@ The build has a number of options that you can learn about using build -?.
 
 See the instructions in [../libraries/webassembly-instructions.md].
 
+### Android
+
+See the instructions in [../../testing/libraries/testing-android.md]
```
Comment (lewing) [inline docs/workflow/building/mono/README.md:58]:
```suggestion
See the instructions for [Testing Android](../../testing/libraries/testing-android.md)
```

**Review by lewing — APPROVED** [review]
Looks good, small markdown suggestion.  Same fix could be made to the wasm link.

File: docs/workflow/building/mono/README.md (hunk 1 above)
Comment (fanyang-mono) [reply docs/workflow/building/mono/README.md:60]: sure

**Pull request merged**